### PR TITLE
M5493 Added tabbed subcollections view

### PR DESCRIPTION
**Pull request by sidohaakma**
- Maybe sum all subcollections samplesizes and present it at the toplevel?

### CODE REVIEW

File: src/components/tables/CollectionsTable.vue [hunk 1]
```diff
@@ -1,50 +1,67 @@
...
-    <tr>
-      <th></th>
-      <th>Collection</th>
-      <th>Type</th>
-      <th>Materials</th>
-      <th>Standards</th>
-      <th>#Samples</th>
-    </tr>
+      <tr>
+        <!--<th></th>-->
```
Comment (Mark-de-Haan) [inline src/components/tables/CollectionsTable.vue]: remove commented code

File: src/components/tables/CollectionsTable.vue [hunk 2]
```diff
@@ -1,50 +1,67 @@
...
-          <i class="fa fa-th fa-lg collection-detail-btn" aria-hidden="true"></i>
-        </a>
-      </td>
+      <template v-for="collection in topLevelElements">
+        <tr>
+          <!--<td :class="{'table-text-content-columns-has-sub': hasSubCollections, 'table-text-content-columns': !hasSubCollections}">-->
+          <!--<a class="btn btn-outline-secondary btn-sm" v-if="hasSubCollections" v-on:click="showSubCollectionsTable">-->
+          <!--<i :class="{'fa fa-plus' : !isSubCollectionsTableVisible, 'fa fa-minus' : isSubCollectionsTableVisible}" aria-hidden="true"></i>-->
+          <!--</a>-->
+          <!--</td>-->
```
Comment (Mark-de-Haan) [inline src/components/tables/CollectionsTable.vue]: Remove commented code

File: src/components/tables/CollectionsTable.vue [hunk 3]
```diff
@@ -1,50 +1,67 @@
...
-        <span v-else-if="column === 'materials'">{{ getCollectionMaterials(collection) }}</span>
-        <span v-else-if="column === 'size'">{{ collection[column] }}</span>
-      </td>
-    </tr>
+            <span v-else-if="column === 'type'">{{ getCollectionType(collection) }}</span>
+            <span v-else-if="column === 'materials'">{{ getCollectionMaterials(collection) }}</span>
+            <span v-else-if="column === 'size'">{{ collection[column] }}</span>
+          </td>
+        </tr>
+        <!--<tr v-if="hasSubCollections && isSubCollectionsTableVisible">-->
```
Comment (Mark-de-Haan) [inline src/components/tables/CollectionsTable.vue]: Remove commented code

File: src/components/tables/CollectionsTable.vue [hunk 4]
```diff
@@ -69,7 +103,20 @@
       },
       getCollectionType (collection) {
         return utils.getUniqueIdArray(collection.type.map(type => type.label)).join(', ')
+      },
+      showSubCollectionsTable () {
```
Comment (Mark-de-Haan) [inline src/components/tables/CollectionsTable.vue]: Remove method together with removing commented code

File: src/components/tables/SubCollectionsTable.vue [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+          <span v-else-if="column === 'type'">{{ getCollectionType(subCollection) }}</span>
+          <span v-else-if="column === 'materials'">{{ getCollectionMaterials(subCollection) }}</span>
+          <span v-else-if="column === 'size'">{{ subCollection[column] }}</span>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</template>
+
+<style>
```
Comment (Mark-de-Haan) [inline src/components/tables/SubCollectionsTable.vue]: Remove commented stylings

Comment (sidohaakma) [reply src/components/tables/SubCollectionsTable.vue]: Is done.

**Pull request merged**